ios/RCTMapboxGL/RCTMapboxAnnotationManager.m

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTMapboxAnnotationManager.h"
+
+#import "RCTUIManager.h"
+#import "RCTConvert+CoreLocation.h"
+#import "UIView+React.h"
+#import "RCTMapboxAnnotation.h"
+
+@interface RCTMapboxAnnotationManager () <MGLMapViewDelegate>
+
+@end
+
+@implementation RCTMapboxAnnotationManager
+
+RCT_EXPORT_MODULE()
+
+- (UIView *)view
+{
+    RCTMapboxAnnotation *marker = [RCTMapboxAnnotation new];
```

Comment:
Nit: `[[RCTMapboxAnnotation alloc] init]`. `new` works, but it’s an antipattern.